from collections import namedtuple

ValidationError = namedtuple("ValidationError", ['message'])    